framework_tvb/tvb/core/entities/storage/datatype_dao.py

```diff
@@ -518,6 +518,10 @@ def get_values_of_datatype(self, project_id, datatype_class, filters=None, page_
                         ).filter(DataType.invalid == False
                         ).filter(or_(Operation.fk_launched_in == project_id,
                                      Links.fk_to_project == project_id))
+
+            if hasattr(datatype_class, 'time_series_type'):
```

Comment:
This if is not sane here.
We should covert the cartesian product with BurstConfig in a left/right outer join on this condition

Reply:
Oh yes, because I changed it in two places actually. The one which is more above in th html file can stay <p> but the second one should be a div, otherwise I the hidden form filters will have to be handled in a different way than the user defined ones (I attached a picture to show what happens when I leave it as a p, basically the way the code is now, when you apply the filters, the effect of the form filters will be ignored). The HTML documentation says that such things (i.e. elements not appearing where you'd expect them) can happen because inline elements are not supposed to contain block elements. 

In conclusion, I either make it a div, or I have to make some changes to the way form filters are handled.

![image](https://user-images.githubusercontent.com/32891492/87921293-07496b00-ca83-11ea-9066-5af970885b5a.png)